app/workers/deletes.php

```diff
@@ -347,4 +353,60 @@ protected function deleteCertificates(Document $document)
             Console::info("No certificate files found for {$domain}");
         }
     }
+
+    protected function deleteBucket(Document $document, string $projectId)
+    {
+        $bucketId = $document->getId();
+        
+        $this->deleteByGroup('files',[
+            new Query('bucketId', Query::TYPE_EQUAL, [$bucketId])
+        ], $this->getInternalDB($projectId));
+
+        $device = new Local(APP_STORAGE_UPLOADS.'/app-'.$projectId);
+        $device->deletePath($device->getRoot() . DIRECTORY_SEPARATOR . $bucketId);
+    }
+    
+    /**
+     * @param string $projectId
+     * @return Database
+     */
+    protected function getInternalDB($projectId): Database
```

Comment:
In the new DB integration we have this methods available in the parent Worker class

Reply:
got it removed!